ods_ci/tests/Tests/100__deploy/130__operators/130__rhods_operator/139__trusted_ca_bundles.robot

```diff
@@ -0,0 +1,155 @@
+*** Settings ***
+Documentation    Test Cases to verify Serverless installation
+Library    Collections
+Resource    ../../../../Resources/Page/OCPDashboard/OCPDashboard.resource
+Suite Setup    Suite Setup
+Suite Teardown    Suite Teardown
+
+
+*** Variables ***
+${RHOAI_OPERATOR_NS}    redhat-ods-operator
+${RHOAI_OPERATOR_DEPLOYMENT_NAME}    rhods-operator
+${TEST_NS}    test-trustedcabundle
+${DSCI_NAME}    default-dsci
+${TRUSTED_CA_BUNDLE_CONFIGMAP}    odh-trusted-ca-bundle
+${CUSTOM_CA_BUNDLE}    test-example-custom-ca-bundle
+
+${IS_PRESENT}    0
+${IS_NOT_PRESENT}    1
+
+
+*** Test Cases ***
+Validate Trusted CA Bundles ConfigMaps
+    [Documentation]  The purpose of this Test Case is to validate the creation of
+    ...    odh-trusted-ca-bundle ConfigMaps
+    [Tags]    Operator
+    ...       ODS-2638
+
+    Log    message=Check that operator is available    level=INFO
+    Is Operator Available    ${RHOAI_OPERATOR_DEPLOYMENT_NAME}    ${RHOAI_OPERATOR_NS}
+    Is DSCI In Ready State    ${DSCI_NAME}    ${RHOAI_OPERATOR_NS}
+
+    Log    message=Validate Trusted CA Bundle Management state Managed    level=INFO
+
+    Create Namespace In Openshift    ${TEST_NS}
+    Wait Until Keyword Succeeds    3 min    0 sec
+    ...    Is Resource Present    project    ${TEST_NS}    ${TEST_NS}    ${IS_PRESENT}
+
+    Wait Until Keyword Succeeds    3 min    0 sec
```

Comment:
It is possible we already have a `Wait Until Resource Is Present` keyword. If not, you could add one in a follow-up PR